packages/playground/src/utils/load_deployment.ts

```diff
@@ -9,21 +9,46 @@ export interface LoadedDeployments<T> {
   items: T[];
 }
 
+interface FailedDeployment {
+  name: string;
+  nodes: number[];
+}
+
 export interface LoadVMsOptions {
   filter?(vm: any): boolean;
 }
 export async function loadVms(grid: GridClient, options: LoadVMsOptions = {}) {
   const machines = await grid.machines.list();
   let count = machines.length;
+  const failedDeployments: FailedDeployment[] = [];
+
+  const promises = machines.map(async (name, index) => {
+    const nodeIds = await grid.machines._getDeploymentNodeIds(name);
+    const machinePromise = grid.machines.getObj(name);
+    const timeoutPromise = new Promise((resolve, reject) => {
+      setTimeout(() => {
+        reject(new Error("Timeout"));
+      }, 10000);
+    });
 
-  const promises = machines.map(name => {
-    return grid.machines.getObj(name).catch(e => {
+    try {
+      const result = await Promise.race([machinePromise, timeoutPromise]);
+      if (result instanceof Error && result.message === "Timeout") {
+        console.log(`%c[Error] Timeout loading deployment with name ${name}`, "color: rgb(207, 102, 121)");
+        return null;
+      } else if ((result as any).length === 0) {
+        console.log(`%c[Error] failed to load deployment with name ${name}}`, "color: rgb(207, 102, 121)");
```

Comment:
`${name}}` remove the extra `}` please 
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/62248851/f9f91723-735e-44f4-ba90-73e132564338)
```suggestion
        console.log(`%c[Error] failed to load deployment with name ${name}`, "color: rgb(207, 102, 121)");
```